src/Microsoft.Diagnostics.Monitoring.WebApi/Models/ProcessIdentifier.cs

```diff
@@ -21,5 +21,8 @@ public class ProcessIdentifier
 
         [JsonPropertyName("name")]
         public string Name { get; set; }
+
+        [JsonPropertyName("is_default")]
```

Comment:
I think the rest of the HTTP surface does not use underscores, so let's call this `isDefault`.